docs/uplink/connect-to-tunnels.md

```diff
@@ -0,0 +1,130 @@
+# Connect to tunnels
+
+The tunnel plugin for the inlets-pro CLI can be used to get connection instructions for a tunnel.
+
+Whether the client needs to be deployed as a systemd service on the customers server or as a kubernetes service, with the cli it is easy to generate connection instructions for these different formats by setting the `--format` flag.
```

Comment:
I don't think you have addressed my comments, but they've disappeared in the rebase perhaps?

cli => CLI
kubernetes => Kubernetes.